File: java/src/main/java/com/factset/protobuf/stach/extensions/models/RowSpanSpread.java
package com.factset.protobuf.stach.extensions.models;

import com.google.protobuf.Value;

public class RowSpanSpread {

    private int position;
    private int rowspan;
    private int colspan;
    private Value value;

    public RowSpanSpread(int position, int rowspan, int colspan, Value value) {
        this.position = position;
        this.rowspan = rowspan;
        this.colspan = colspan;
        this.value = value;
    }

    public int getPosition() {
        return position;
    }

    public int getRowspan() {
        return rowspan;
    }

    public int getColspan() {
        return colspan;
    }

    public Value getValue() {
        return value;
    }

    public void setPosition(int position) {
        this.position = position;
    }

    public void setRowspan(int rowspan) {
        this.rowspan = rowspan;
    }

    public void setColspan(int colspan) {
        this.colspan = colspan;
    }

    public void setValue(Value value) {
        this.value = value;
    }
}

File: dotnet/StachExtensions/FactSet.Protobuf.Stach.Extensions/Models/RowSpanSpread.cs
using Google.Protobuf.WellKnownTypes;

namespace FactSet.Protobuf.Stach.Extensions.Models
{
    public class RowSpanSpread
    {
        public RowSpanSpread(int position, int rowSpan, int colSpan, Value value)
        {
            Position = position;
            RowSpan = rowSpan;
            ColSpan = colSpan;
            Value = value;
        }
        
        /// <summary>
        /// Position at which the value needs to be added.
        /// </summary>
        public int Position { get; set; }
        
        /// <summary>
        /// The number of rows across which the value has to be spread.
        /// </summary>
        public int RowSpan { get; set; }
        
        /// <summary>
        /// The number of columns across which the value has to be spread.
        /// </summary>
        public int ColSpan { get; set; }
        
        /// <summary>
        /// The value that is to be spread.
        /// </summary>
        public Value Value { get; set; }
    }
}